tests/Integration/Behaviour/Features/Context/Domain/CustomerServiceFeatureContext.php

```diff
@@ -65,11 +83,13 @@ public function createCustomerThread(string $threadReference, TableNode $table):
     {
         $data = $table->getRowsHash();
 
+        $contactReference = $this->contactRegistry[$data['contactReference']];
```

Comment:
```suggestion
        $contactId = $this->getSharedStorage()->get($data['contactReference']);
```

Reply:
Or even better, we introduce this helper since we do this operation very often
```suggestion
        $contactReference = $this->referenceToId($data['contactReference']);
```